examples/calibration/plot_calibration.py

```diff
@@ -10,16 +10,17 @@
 while others being under-confident. Thus, a separate calibration of predicted
 probabilities is often desirable as a postprocessing. This example illustrates
 two different methods for this calibration and evaluates the quality of the
-returned probabilities using Brier's score
-(see https://en.wikipedia.org/wiki/Brier_score).
+returned probabilities using a calibration loss defined as the average
+absolute deviation between the calibration curve and the diagonal (perfectly
+calibrated classifier).
 
 Compared are the estimated probability using a Gaussian naive Bayes classifier
 without calibration, with a sigmoid calibration, and with a non-parametric
 isotonic calibration. One can observe that only the non-parametric model is
```

Comment:
Would you be able to expand on the 'empirical' line in the the graph as well?

Reply:
What would you add here specifically ?
By the way, as for your previous comment, those parts of the doc are not from this PR, so not sure what's the best practice is...

Reply:
Nevermind, since it wasn't changed in this PR.